server/views/pages/teamOverview.njk

```diff
@@ -263,6 +264,30 @@
     </div>
   {% endif %}
 
+  <h3 class="govuk-heading-m">Component Drift Radiator</h3>
+
+{% if team.component.length > 0 %}
+  {{ driftRadiatorTable() }}
+{% else %}
+  {{ "This team has no products to display" }}
```

Comment:
Don't need `{{ "` as it's just text? Just needs to be normal HTML so `<p>....</p>`